source/modules/MEI.shared.xml

```diff
@@ -3559,6 +3565,40 @@
         C to C♯ and one from C to D♭.</p>
     </remarks>
   </classSpec>
+  <classSpec ident="att.tuning.log" module="MEI.shared" type="atts">
+    <desc xml:lang="en">Logical domain attributes for tuning.</desc>
+    <attList>
+      <attDef ident="tuning.standard" usage="opt">
+        <desc xml:lang="en">Describes the tuning standard used.</desc>
+        <datatype>
+          <rng:data type="NMTOKEN"/>
+        </datatype>
+        <valList type="semi">
+          <valItem ident="guitar.standard">
+            <desc xml:lang="en">Standard tuning for current guitars. The courses are tuned to E2 A2 D3 G3 B3 E4.</desc>
+          </valItem>
+          <valItem ident="guitar.drop.D">
+            <desc xml:lang="en">Drop D tuning for guitars. The lowest course is tuned down to D, while all other courses are kept to their regular pitches. D2 A2 D3 G3 B3 E4.</desc>
+          </valItem>
+          <valItem ident="guitar.open.D">
+            <desc xml:lang="en">Open D tuning for guitars. D2 A2 D3 F3s A3 D4.</desc>
+          </valItem>
+          <valItem ident="guitar.open.G">
+            <desc xml:lang="en">Open G tuning for guitars. D2 G2 D2 G2 B3 D4.</desc>
+          </valItem>
+          <valItem ident="guitar.open.A">
+            <desc xml:lang="en">Open A tuning for guitars. E2 A2 E3 A3 C4s E4.</desc>
+          </valItem>
+          <valItem ident="lute.renaissance.6">
+            <desc xml:lang="en">Renaissance tuning for lutes with 10 strings on 6 courses. G2G3 C3C4 F3F4 A4A4 D4 G4.</desc>
```

Comment:
I get that it's tuned in octaves here, but see my comment above on grouping courses in the 12-string guitar example. This seems to be another way of communicating it which isn't consistent.